### DISCUSSION TITLE
只要评论有图片，后台的进入就非常缓慢，甚至无法进入

### DISCUSSION THREAD

**Discussion by YuukiRei** [walinejs/waline#931] [Q&A] [answered]
有没有办法规避这种情况呢😖
数据库是腾讯云的数据库
服务端部署在vercel

**Comment by YuukiRei** [comment, marked as answer]
或者有没有限制图片大小或者禁止图片评论的方法呢😶‍🌫️

**Reply by lizheming** [reply]
可以自己自定义 uploadImage 实现的

**Reply by lizheming** [reply]
禁止上传的话直接 `uploadImage: function() {}` 就可以了

**Reply by YuukiRei** [reply]
👌了解啦感谢

**Comment by Mister-Hope** [comment]
这里我已经在群里跟 @lizheming  说过了，我个人认为默认情况下把图片 base64 编入评论是非常不合理的。

这样用户可以在 IPQPS 设置很低的情况下，也能轻易炸库。

你等等看看他怎么回把。

**Reply by lizheming** [reply]
base64 编入没什么问题，可以丰富下默认的 uploadImage，增加大小的限制

**Reply by Mister-Hope** [reply]
我感觉性能问题会很严重
